<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_connect</class>
 <widget class="QDialog" name="Dialog_connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="car_con_lab">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>車子連接中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wait_dia_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_car_loading_gif">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
